packages/babel-preset-default/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Added `addPolyfillComments` option. When used, it will automatically add magic comments to mark files that need `wp-polyfill`.
```

Comment:
This needs a change type header like `### Internal`.

Does this change the package's behavior in any significant ways for 3rd parties?

Reply:
It changes behaviour in the sense that, when the `addPolyfillComments` option is enabled, any code that goes through this babel config will end up with magic comments. So in that sense it becomes part of the public API, even though its intended use is only internal.

Do you think the option's name itself should reflect its internal-ness?

Reply:
`### Internal` was only intended as an example, not a suggestion. We don't need to change the option name, but since this really isn't intended for third parties at the moment, it can go under an `Internal` header.

Reply:
Done!